### PR TITLE
Updated Bugs and Textures

### PR DESCRIPTION
**Pull request by Abdullah-coder2013**
## Pull Request Summary

I was trying to solve the issue with textures not loading and player doesn't sneak, jump or sprint.
Then solved the texture loading issue by taking everything out of the `src` directory.
As for the actions issue, I deleted the `firstpersoncontroller` file and exchanged it with the built-in `Firstpersoncontroller` class from `ursina`.

## Pull Request Info

This includes new textures for the new block model.

## Validation Steps Performed

Just run the `main.py` file in the root directory.

### CODE REVIEW

File: src/player/arm.py
Comment (CMihai99) [inline src/player/arm.py:1]: Why did you delete the arm file? Will the arm still show in-game?

File: src/player/actions/jumping.py
Comment (CMihai99) [inline src/player/actions/jumping.py:1]: Why did you delete the jumping file? I also see that in `src/player/actions/jumping.py` you removed the `jumping` import component on (old) line 19, aswell as the declared `jumping` import component on (old) line 24.

File: main.py
Comment (CMihai99) [inline main.py:1]: Why did you remove the `jumping` import component on ***(old)* line 19**, as well as the declared `jumping` import component on ***(old)* line 24** in the `main.py` file?

**Review by CMihai99 — APPROVED** [review]
Some questions addressing this pull:
1. Why did you delete the `src/player/actions/jumping.py` file?
2. Why did you remove the `jumping` import component on ***(old)* line 19**, as well as the declared `jumping` import component on ***(old)* line 24** in the `main.py` file?

File: src/player/actions/jumping.py
Comment (Abdullah-coder2013) [reply src/player/actions/jumping.py:1]: I deleted it because we don’t need it anymore.
The `Firstpersoncontroller` class already has jumping built in

File: main.py
Comment (Abdullah-coder2013) [reply main.py:1]: I deleted both of them see line **_(new)_ line 19** and **line 24**
